osutil/disks/disks_linux.go

```diff
@@ -345,52 +346,70 @@ func diskFromMountPointImpl(mountpoint string, opts *Options) (*disk, error) {
 		if err != nil {
 			return nil, fmt.Errorf(errFmt, partMountPointSource, err)
 		}
+		dmUUID = bytes.TrimSpace(dmUUID)
 
 		dmName, err := ioutil.ReadFile(filepath.Join(dmDir, "name"))
 		if err != nil {
 			return nil, fmt.Errorf(errFmt, partMountPointSource, err)
 		}
+		dmName = bytes.TrimSpace(dmName)
```

Comment:
FYI, this code conflicts with my multi-volume gadget changes in https://github.com/snapcore/snapd/pull/10919

Reply:
But the conflict can be resolved obviously, it's not a fundamental conflict, just something to keep in mind as we land things